src/commands/server/start.ts

```diff
@@ -186,43 +188,8 @@ export default class Start extends Command {
     })
   }
 
-  setPlaformDefaults(flags: any) {
-    if (flags.platform === 'minishift') {
-      if (!flags.multiuser && flags.installer === '') {
-        flags.installer = 'minishift-addon'
-      }
-      if (flags.multiuser && flags.installer === '') {
-        flags.installer = 'operator'
-      }
-    } else if (flags.platform === 'minikube') {
-      if (!flags.multiuser && flags.installer === '') {
-        flags.installer = 'helm'
-      }
-      if (flags.multiuser && flags.installer === '') {
-        flags.installer = 'operator'
-      }
-    } else if (flags.platform === 'openshift') {
-      if (flags.installer === '') {
-        flags.installer = 'operator'
-      }
-    } else if (flags.platform === 'k8s') {
-      if (flags.installer === '') {
-        flags.installer = 'helm'
-      }
-    } else if (flags.platform === 'docker-desktop') {
-      if (flags.installer === '') {
-        flags.installer = 'helm'
-      }
-    } else if (flags.platform === 'crc') {
-      if (flags.installer === '') {
-        flags.installer = 'operator'
-      }
-    }
-
-    // TODO when tls by default is implemented for all platforms, make `tls` flag turned on by default.
-    if (flags.installer === 'helm' && (flags.platform === 'k8s' || flags.platform === 'minikube' || flags.platform === 'microk8s')) {
```

Comment:
if helm is used for OpenShift instance ?

Reply:
The behaviour will be the same as before:
```
 ›   Error: Error: 🛑 Current platform is crc. Helm installer is only available on top of Kubernetes flavor platform (including Minikube, Docker Desktop).
```